service_integration/recruitment_service_integration.js

```diff
@@ -0,0 +1,19 @@
+const fetch = require("node-fetch")
+
+const serviceUrl = "https://recruitment-service-estella.herokuapp.com/"
+
+
+
+const recruitmentServiceVeirfy = async (token) => {
+    const response = await fetch(serviceUrl + "api/users/loggedInUser", {
+        method: 'GET',
+        headers: {
+            'Content-Type': 'application/json',
+            'x-jwt': token
+        }
+    })
+    const jsonResponse = await response.json()
+    return "statusCodeValue" in jsonResponse ? null : jsonResponse.mail
```

Comment:
hmmm?

Reply:
Taka dziwna rzecz że jak dostaniesz response z błędem z textem zamiast body to tak wygląda status bo sam status to zawsze 200 czy jakoś tak

Reply:
XD, w sensie ufam, że to działa, ale nie wiedziałem właśnie o co tu chodzi XDDD